### DISCUSSION TITLE
AxisVertical: Inner Top label on position 0

### DISCUSSION THREAD

**Discussion by Antiglobalist** [patrykandpatrick/vico#697] [Ideas]
Hi! I have created PR that provides ability to add Label to 0 position inside the chart 
What do you think about it? Can somebody review it ?

I need it in my pet project. 
Or if you know some easy way to implement it without updating SDK - pls let me know how 

https://github.com/patrykandpatrick/vico/pull/679

**Comment by Gowsky** [comment]
Hello, yes, we will review your PR as soon as possible.

In the meantime you can create `VerticalAxis` subclass, or use `HorizontalLine`.

**Comment by Gowsky** [comment]
This has been addressed in [Vico 2.3.3](https://github.com/patrykandpatrick/vico/releases/tag/v2.3.3).
